package-res/resources/web/pentaho/util/arg.js

```diff
@@ -20,23 +20,73 @@ define([
 
   var A_slice = Array.prototype.slice;
 
+  /**
+   * @name pentaho.util.arg
+   * @class
+   * @amd pentaho/util/arg
+   *
+   * @classDesc Util methods for keyword args.
+   *
+   * The AMD module returns the arg utils.
+   *
+   * @description Performs actions on keyword args.
+   */
   return {
+    /**
+     * Provides the optional property value of a given object or the given default value if the given object does
+     * not contain the sought after property value.
+     *
```

Comment:
This formulation is correct but it seems overly complex. Can you break it down to a simpler formulation?